src/lv_widgets/lv_spinbox.h

```diff
@@ -135,6 +135,20 @@ bool lv_spinbox_get_rollover(lv_obj_t * spinbox);
  */
 int32_t lv_spinbox_get_value(lv_obj_t * spinbox);
 
+/**
+ * Get the spinbox step value (user has to convert to float according to its digit format)
+ * @param spinbox pointer to spinbox
+ * @return value integer step value of the spinbox
+ */
+static inline int32_t lv_spinbox_get_step(lv_obj_t * spinbox)
+{
+    LV_ASSERT_OBJ(spinbox, LV_OBJX_NAME);
```

Comment:
Because `LV_OBJX_NAME` can't be defined in a header file this assert causes build errors. I would just remove it.

Reply:
Ahh perfect, thanks I have now removed that